components/script/dom/bindings/codegen/Bindings.conf

```diff
@@ -40,7 +40,7 @@ DOMInterfaces = {
 
 'Range': {
     'weakReferenceable': True,
-    'canGc': ['CreateContextualFragment', 'ExtractContents', 'CloneContents', 'CloneRange', 'SurroundContents'],
+    'canGc': ['CreateContextualFragment', 'ExtractContents', 'CloneContents', 'CloneRange', 'SurroundContents', 'ParseFromString', 'GetBounds', 'GetClientRects'],
```

Comment:
Sorry, I wasn't clear. Each canGc section in this file applies to a single interface name. You will need to add new sections for each interface you want to change in this way, then run a build to verify that the arguments are now present, and then add the missing arguments once the build fails.

Reply:
okay, i think i understand! so the `'weakReferenceable': True,` attribute should be there for all the new sections, right?

Reply:
No. Any interface that is not already present can have a new section added with only the canGc property present.

Reply:
also, is this a valid section relating to the changes in `domrect.rs`?
`'DOMRect': {
    'weakReferenceable': True,
    'canGc': ['new'],
},`

Reply:
Ah, no, it's only for methods that appear in the DOMRectMethods trait in this case.

Reply:
Can you please check if the changes are correct this time?